floor/config/midi_maps/akai-apcmini.json

```diff
@@ -29,6 +29,34 @@
       "command": ["control_mode_change", 56],
       "function": "set_brightness"
     },
+    {
+      "command": ["control_mode_change", 48],
+      "function": "ranged_value_1"
+    },
+    {
+      "command": ["control_mode_change", 49],
+      "function": "ranged_value_2"
+    },
+    {
+      "command": ["control_mode_change", 50],
+      "function": "ranged_value_3"
+    },
+    {
+      "command": ["control_mode_change", 51],
+      "function": "ranged_value_4"
+    },
+    {
+      "command": ["control_mode_change", 52],
+      "function": "ranged_value_5"
```

Comment:
nb i think you can drop mappings for 5,6,7 now

Reply:
Whoops, missed that.